examples/algo-wallet-demo/index.js

```diff
@@ -101,6 +102,15 @@ class App extends React.Component {
     this.appendMsg('Using PeraConnect');
   }
 
+  async useAlgoSignerConnect() {
+    delete window.algorand;
+    stdlib = loadStdlib(process.env);
+    stdlib.setWalletFallback(reach.walletFallback({
+      // ...we use a different fallback here:
+      providerEnv, MyAlgoConnect: MakeAlgoSignerConnect(AlgoSigner,'TestNet') }));
```

Comment:
I am ignorant... will this error during loading on browsers that don't have AlgoSigner, or will it not because it won't actually evaluate this line?

Reply:
Those lines wouldn't be evaluated during page load, but it would trigger an error if a user clicks the "Use AlgoSignerConnect" button and there is no AlgoSigner object.

If we want to ensure that walletFallback doesn't fail (and leave the failing to a function like .getDefautAccount) we could have the AlgoSignerConnect class assume the presence of the AlgoSigner object without passing it to the `MakeAlgoSignerConnect` function as an argument.

So either this:

```
export default function ALGO_MakeAlgoSignerConnect( provider: string = 'TestNet' ) {
    return class AlgoSignerConnect {
        constructor() {
        }

        async connect() {
            //@ts-ignore
            await AlgoSigner.connect();
            //@ts-ignore
            return await AlgoSigner.accounts({ledger:provider});
        }
// ...
```

Or using the class' constructor to grab the AlgoSigner object instead, i.e.:

```
export default function ALGO_MakeAlgoSignerConnect( provider: string = 'TestNet' ) {
    return class AlgoSignerConnect {
        AlgoSigner: any = null;
        
        constructor() {
            //@ts-ignore
            if (typeof AlgoSigner != 'undefined') this.AlgoSigner = AlgoSigner;
        }
// ... and change all other references from AlgoSigner to this.AlgoSigner
```

In either case, walletFallback would change to the following:

```
stdlib.setWalletFallback(reach.walletFallback({
      providerEnv, MyAlgoConnect: MakeAlgoSignerConnect('TestNet') }));
```

Let me know if that's a preferable approach.. or if you think a different angle would be better.